src/components/ProjectsPage/CreateProjectDialog/CreateProjectDialog.js

```diff
@@ -115,10 +116,10 @@ const CreateProjectDialog = ({
                   onClick={closeNewProjectPopUp}
                 />
                 <Button
-                  disabled={projectStore.loading || !formState.values.name || formState.invalid}
+                  disabled={projectStore.loading || (formState.invalid && formState.submitFailed)}
```

Comment:
Probably there should be a condition for `formState.submitting` as in other Wizards

Reply:
Also, better to create a functions that returns this condition and use it in all places when we need it.